src/alfasim_sdk/_internal/hook_specs_gui.py

```diff
@@ -194,12 +202,11 @@ def alfasim_get_status(
         from alfasim_sdk import Quantity
         from alfasim_sdk import ErrorMessage
 
+
         # Define MyModel used in this plugin
         @data_model(icon="", caption="My Plugin Model")
         class MyModel:
-            distance = Quantity(
-                value=1, unit="m", caption="Distance"
-            )
+            distance = Quantity(value=1, unit="m", caption="Distance")
```

Comment:
I just ask myself why did it reformat the string.

Reply:
It also formats code in docstrings IIRC.